volume/local/local.go

```diff
@@ -334,6 +334,11 @@ func (v *localVolume) Path() string {
 	return v.path
 }
 
+// CachedPath returns the data location
+func (v *localVolume) CachedPath() string {
```

Comment:
where is this used ?

Reply:
Just above `setBindModeIfNull`. The path must be set otherwise relabeling will fail.